test/TestApps/ToastNotificationsTestApp/main.cpp

```diff
@@ -1288,6 +1288,34 @@ bool VerifyRemoveAllAsync()
     return true;
 }
 
+
+bool VerifyForegroundRegistrationSuccess()
```

Comment:
nit: looks like this is to verify the foreground registration succeed after un-registration. Maybe we can make this name of this test more descriptive.